ddtrace/settings/config.py

```diff
@@ -212,9 +216,12 @@ def __init__(self):
         self.env = os.getenv("DD_ENV") or self.tags.get("env")
         self.service = os.getenv("DD_SERVICE", default=self.tags.get("service", DEFAULT_SPAN_SERVICE_NAME))
 
-        if self.service is None and in_gcp_function():
+        if self.service is None and self._is_gcp_function:
```

Comment:
I think it's debatable whether these flags belong in `config` or in a servless utils file. I don't have a strong opinion about that but I wanted to raise it for your awareness.

Reply:
Go ahead and move them back to function calls. I think that's more appropriate in this context since running in a GCP function isn't a "configuration" in the way the library uses that term.

Reply:
switched over!